extensions/google-cloud-functions-http/runtime/src/main/java/io/quarkus/gcp/functions/http/QuarkusHttpFunction.java

```diff
@@ -40,6 +40,12 @@ public class QuarkusHttpFunction implements HttpFunction {
 
     private static final int BUFFER_SIZE = 8096;
 
+    private static final String USER_AGENT = "User-Agent";
+    private static final String USER_AGENT_VALUE = String.format(
```

Comment:
Don't store it in a static var as this will reflect the build environment, not the runtime

Reply:
Ok , I will remove this.